static/app/components/replaysOnboarding/utils.tsx

```diff
@@ -30,3 +33,16 @@ export function splitProjectsByReplaySupport(projects: Project[]) {
 export const replayJsFrameworkOptions: PlatformIntegration[] = platforms.filter(p =>
   replayFrontendPlatforms.includes(p.id)
 );
+
+export const tracePropagationMessage = (
+  <Alert type="info" showIcon>
+    {tct(
+      `To see replays for backend errors, ensure that you have set up trace propagation. To learn more, [link:read the docs].`,
+      {
+        link: (
+          <ExternalLink href="https://docs.sentry.io/product/session-replay/getting-started/#:~:text=Make%20sure%20you%27ve%20set%20up%20trace%20propagation%20in%20your%20backend%20projects." />
```

Comment:
What do you think about linking to the section title instead? 
`https://docs.sentry.io/product/session-replay/getting-started/#replays-for-backend-errors`

I like when the link text appears rigth on the new page, which is happening already. The prompt here is "To see replays for backend errors,... " and the title highlighted text would be "trace propagation in your backend projects"

But that only works for Chrome users. The section title is "Replays for Backend Errors", so I think it's still a good link target, and would be more cross-browser.

Reply:
makes sense, i'll update!